docs/programming-oak.md

```diff
@@ -379,20 +379,26 @@ each chat room and saves off the write handle that will be used to send messages
 to the room:
 
 <!-- prettier-ignore-start -->
-[embedmd]:# (../examples/chat/module_0/rust/src/lib.rs Rust /.*Create a new Node/ /^ +\);$/)
+[embedmd]:# (../examples/chat/module_0/rust/src/lib.rs Rust /.*create_chat_room()/ /\}$/)
```

Comment:
This comment is showing in the HTML output &ndash; I think the parentheses need escaping.